src/reducers/console.js

```diff
@@ -28,6 +28,11 @@ export default function console(stateIn, {type, payload, meta}) {
         meta.key,
         new ConsoleEntry({expression: payload}),
       );
+    case 'CONSOLE_LOG_PRODUCED':
+      return state.set(
+        payload.key,
+        new ConsoleEntry({expression: '', value: payload.value}),
```

Comment:
No need to pass an explicit `expression` here, can just use the record’s default (which as mentioned elsewhere I think should change to `null`)